### ISSUE TITLE
Missing dependency "byebug"

### ISSUE THREAD

**Issue by lushc** [itay-grudev/capistrano-env_config#1]
```
LoadError: cannot load such file -- byebug
/Users/chris/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/capistrano-env_config-0.0.3/lib/capistrano/tasks/env_config.rake:1:in `require'
/Users/chris/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/capistrano-env_config-0.0.3/lib/capistrano/tasks/env_config.rake:1:in `eval_rakefile'
/Users/chris/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/capistrano-3.10.1/lib/capistrano/plugin.rb:88:in `instance_eval'
/Users/chris/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/capistrano-3.10.1/lib/capistrano/plugin.rb:88:in `eval_rakefile'
/Users/chris/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/capistrano-env_config-0.0.3/lib/capistrano/env_config.rb:7:in `define_tasks'
/Users/chris/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/capistrano-3.10.1/lib/capistrano/configuration/plugin_installer.rb:27:in `install'
/Users/chris/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/capistrano-3.10.1/lib/capistrano/configuration.rb:155:in `install_plugin'
/Users/chris/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/capistrano-env_config-0.0.3/lib/capistrano/env_config.rb:16:in `<top (required)>'
```

**Comment by itay-grudev** [comment]
OMG! I've left a byebug statement in the library?!

**Comment by itay-grudev** [comment]
I'll fix it by 9pm UTC.

**Comment by itay-grudev** [comment]
@lushc Because this is new library I made and I have absolutely no feedback, what do you think? And proposals for improvements?

**Comment by lushc** [comment]
Thanks, no need to rush though 😄  I haven't had much of a chance to properly test it yet, just exploring ways of managing env across deploys, but on the surface it does seem to be useful! When I have it integrated into our workflow I'll have some better feedback.

**Comment by itay-grudev** [comment]
Solved in `v0.0.4`.

**Comment by itay-grudev** [comment]
P.S. For what it's worth I've been using it on staging and production on two projects for several months now without any problems.
